styles.css

```diff
@@ -709,6 +709,114 @@ footer hr.footer-social-network {
   padding-left: 10px;
 }
 
+.modal-container {
```

Comment:
>  _This comment refers to line #381_
- [ ] [**_OPTIONAL_**]I think it would be nice to show `see project` button when hovering on project cards rather than rotating the card which makes it difficult to click on as shown below:
![Screenshot from 2021-12-15 21-17-58](https://user-images.githubusercontent.com/61377322/146244342-ede9b7a0-2074-41fd-b07b-639d76314c67.jpg)